CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## v0.4.1 - 2020-09-15
+
+This is a patch release that mainly includes bug fixes.
+
+### Changes in v0.4.1
+
+#### Features
+
+- feat: add severity labels to MetalLB alerts ([#925](https://github.com/kinvolk/lokomotive/pull/925)).
+
+#### Bug fixes
+
+- Override memory limits of rook operator to 512Mi ([#938](https://github.com/kinvolk/lokomotive/pull/938)).
+- Fix envoy grafana dashboard errors ([#969](https://github.com/kinvolk/lokomotive/pull/969)).
+- MetalLB: Fix regressions of tolerations and nodeSelectors ([#927](https://github.com/kinvolk/lokomotive/pull/927)).
+- Fix controlplane components order ([#937](https://github.com/kinvolk/lokomotive/pull/937)).
+- component/metallb: Fix controller tolerations ([#931](https://github.com/kinvolk/lokomotive/pull/931)).
+
+#### Docs
+
+- docs: fix etcd version upgrade sed expression ([#921](https://github.com/kinvolk/lokomotive/pull/921)).
+- docs: fix rook version update command ([#930](https://github.com/kinvolk/lokomotive/pull/930)).
+
+#### Development
+
+- Fix output of `convertNodeSelector` in rook ([#945](https://github.com/kinvolk/lokomotive/pull/945)).
+- Dex convert to helm chart and update to v2.25.0 ([#962](https://github.com/kinvolk/lokomotive/pull/962)).
+- httpbin convert to helm chart ([#965](https://github.com/kinvolk/lokomotive/pull/965)).
+- FLUO: convert to Helm chart ([#935](https://github.com/kinvolk/lokomotive/pull/935)).
+- Makefile: Don't build before linting ([#901](https://github.com/kinvolk/lokomotive/pull/901)).
```

Comment:
the PR actually adds a separate target to run linter without building.

Reply:
But this is still development and not for the users, but for the devs. I have added a line about new target. But this should still remain in this section.

Reply:
Yes, it's fine. Just the title was wrong.